docs/installation/compose.md

```diff
@@ -0,0 +1,221 @@
+
+
+DJANGO_EMAIL_BRAND_NAME=<brand name used in email templates> # e.g. "La Suite Numérique"
+DJANGO_EMAIL_LOGO_IMG=<logo image to use in email templates.> # e.g. "https://docs.yourdomain.tld/assets/logo-suite-numerique.png" 
+```
+
+### AI
+
+Built-in AI actions let users generate, summarize, translate, and correct content.
+
+AI is disabled by default. To enable it, the following environment variables must be set in in `env.d/backend`:
+
+```env
+AI_FEATURE_ENABLED=true # is false by default
+AI_BASE_URL=https://openaiendpoint.com
+AI_API_KEY=<API key>
+AI_MODEL=<model used> e.g. llama
+```
+
+### Frontend theme
+
+You can [customize your Docs instance](../theming.md) with your own theme and custom css.
+
+The following environment variables must be set in `env.d/backend`:
+
+```env
+FRONTEND_THEME=default # name of your theme built with cuningham
+FRONTEND_CSS_URL=https://storage.yourdomain.tld/themes/custom.css # custom css
+```
+
+## Step 3: Reverse proxy and SSL/TLS
+
+> [!WARNING]
+> In a production environment, configure SSL/TLS termination to run your instance on https.
+
+If you have your own certificates and proxy setup, you can skip this part.
+
+You can follow our [nginx proxy example](../examples/compose/nginx-proxy/README.md) with automatic generation and renewal of certificate with Let's Encrypt. 
+
+You will need to uncomment the environment and network sections in compose file and update it with your values.
+
+```yaml
+  frontend:
+    ...
+    # Uncomment and set your values if using our nginx proxy example
+    #environment:
+    # - VIRTUAL_HOST=${DOCS_HOST} # used by nginx proxy 
+    # - VIRTUAL_PORT=8083 # used by nginx proxy
+    # - LETSENCRYPT_HOST=${DOCS_HOST} # used by lets encrypt to generate TLS certificate
+    ...
+# Uncomment if using our nginx proxy example
+#    networks:
+#    - proxy-tier
+#
+#networks:
+#  proxy-tier:
+#    external: true
+```
+
+## Step 4: Start Docs
+
+You are ready to start your Docs application !
+
+```bash
+docker compose up -d
+```
+> [!NOTE]
+> Version of the images are set to latest, you should pin it to the desired version to avoid unwanted upgrades when pulling latest image.
+
+## Step 5: Run the database migration and create Django admin user
+
+```bash
+docker compose run --rm app python manage.py migrate
+docker compose run --rm app python manage.py createsuperuser --email <admin email> --password <admin password>
+```
+
+Replace `<admin email>` with the email of your admin user and generate a secure password. 
+
+Your docs instance is now avalailable on the domain you defined, https://docs.yourdomain.tld.
+
+THe admin interface is available on https://docs.yourdomain.tld/admin with the admin user you just created.
+
+## How to upgrade your Docs application
+
+### Step 1: Edit the images tag with the desired version
+
+### Step 2: Pull the images
+
+```bash
+docker compose pull
+```
+
+### Step 3: Restart yout containers
```

Comment:
Before restarting the containers you should suggest to read the `UPGRADE.md` file.